src/main/java/com/esri/core/geometry/OperatorExportToEWkbLocal.java

```diff
@@ -0,0 +1,36 @@
+package com.esri.core.geometry;
+
+import java.nio.ByteBuffer;
+import java.nio.ByteOrder;
+
+import static com.esri.core.geometry.OperatorExportToWkbLocal.exportToWKB;
```

Comment:
Explicitly call the method instead of import static. It makes the code more readable.

Reply:
https://github.com/Esri/geometry-api-java/pull/271/commits/b0715e090d6c2b2bc20dff4636549a5ebc6e3ccc#diff-0ac3fd646409d9bbd15657dfdcb45878a1daf3da65607da6906502c0a7ffa979R32